tests/suites/storage/charm_storage.sh

```diff
@@ -0,0 +1,142 @@
+run_charm_storage() {
+	echo
+
+	model_name="test-storage"
```

Comment:
here we need to use `ensure` see the other examples: 

https://github.com/juju/juju/blob/423338a5ae96cb7ee7c678d25f8f6629922802ac/tests/suites/deploy/deploy_charms.sh#L6

Reply:
I had it then removed it because I noticed we are already creating a model in the `task.sh` when we run `bootstrap ${model_name} ${file}`. Ensure then again creates another model. And since this test only has one subtest. I reckon that there is no need to create another model instead it can just use the already created model.

Reply:
EDIT: I had not considered there might be need to run this subtests individually hence the need for `ensure` . Thank you for catching this. Will add it.